types/repo/AnonymousMessageRepo/AnonymousPostNotificationData.ts

```diff
@@ -19,6 +19,10 @@ export interface PostMaker {
 export interface PostMakerData {
   profile_pic_url?: string;
   username: string;
+  emoji_name: string | null;
+  emoji_code: string | null;
+  color_name: string | null;
+  color_code: string | null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new fields `emoji_name`, `emoji_code`, `color_name`, and `color_code` have been added to the `PostMakerData` interface. Ensure that these fields are properly handled in all places where this interface is used, including proper null checks since these fields can be null. If not handled correctly, it could lead to runtime errors. 

Also, consider adding comments to describe what these fields represent, especially when they can be null. This will improve code readability and maintainability.


<!-- This is an auto-generated comment by OSS CodeRabbit -->